gradle.properties

```diff
@@ -1,7 +1,7 @@
 org.gradle.daemon=false
 org.gradle.parallel=false
 org.gradle.jvmargs=-Xms512m -Xmx512m
-scalaVersion=2.12.10
+scalaVersion=2.13.6
```

Comment:
Should we update the following version in `kafka-cruise-control-start.sh`?
```
if [ -z "$SCALA_VERSION" ]; then
  SCALA_VERSION=2.12.10
fi
```

Reply:
Yes we should! (`grep -r` FTW)!